includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -992,7 +994,9 @@ public function generate_upe_checkout_experience_accepted_payments_html( $key, $
 
 		foreach ( $this->payment_methods as $method_id => $method ) {
 			$method_enabled       = in_array( $method_id, $this->get_upe_enabled_payment_method_ids(), true ) ? 'enabled' : 'disabled';
-			$subtext_messages     = $method->get_subtext_messages();
+			$capability_id        = "{$method_id}_payments"; // "_payments" is a suffix that comes from Stripe API, except when it is "transfers", which does not apply here
+			$method_status        = isset( $stripe_capabilities[ $capability_id ] ) ? $stripe_capabilities[ $capability_id ] : 'inactive';
```

Comment:
(minor) I wonder if we should show the message at all when the payment method is not set in the capabilities (or when the account is not connected) 🤔

Reply:
Considering that on both cases we would still have a "pending activation" status per se, I think it's safe to assume it'd be better than have nothing indicating that.